### ISSUE TITLE
Are the HUC8s simplified?

### ISSUE THREAD

**Issue by brucecrevensten** [ua-snap/geospatial-vector-veracity#107]
The HUC8s are ~36M in this repo -- have these been simplified? 

`vector_data/polygon/boundaries/alaska_hucs/*`

**Comment by kyleredilla** [comment]
I don't believe they have been, just modified to remove some columns etc. Should they be simplified?
